client/src/components/Kanban/Board.tsx

```diff
@@ -1,17 +1,24 @@
 import { Grid } from '@material-ui/core';
-import { DragDropContext } from 'react-beautiful-dnd';
+import { DragDropContext, Droppable, DroppableProvided } from 'react-beautiful-dnd';
 import { useKanban } from '../../context/useKanbanContext';
 import Column from './Column/Column';
 
 const Board = (): JSX.Element => {
   const { columns, handleDragEnd } = useKanban();
   return (
     <DragDropContext onDragEnd={handleDragEnd}>
-      <Grid container>
-        {columns.map((column) => (
-          <Column key={column.id} id={column.id} name={column.name} cards={column.cards} />
-        ))}
-      </Grid>
+      <Droppable droppableId="board" type="column" direction="horizontal">
```

Comment:
Conditionally setting direction to vertical on small screens and horizontal on larger screens will fix the issue with dragging the columns vertically. However, this is tricky on medium sized screens where the layout is 2 columns wide.

Reply:
You might be able to alleviate this issue a bit by making the columns get smaller to a breakpoint on medium screens.